src/components/ui/Button.jsx

```diff
@@ -2,7 +2,7 @@
 const Button = () => {
     return (
     <button>
-        some changes
+        HELLO WORLD
     </button>
     )
     
```

Comment:
Remove this, this is unnecessary or whatever.

Reply:
@SAMAHANSysDev